charts/amoro/templates/_optimizer.tpl

```diff
@@ -84,3 +84,13 @@ properties:
   {{- end -}}
 {{- end -}}
 
+{{- define "amoro.optimizer.container.k8s" -}}
+container-impl: com.netease.arctic.server.manager.KubernetesOptimizerContainer
+properties:
+  kube-config-path: "～/.kube/config"
+  image: arctic163/amoro:0.6
```

Comment:
Template the image information from `values.yml` 

Or you can implement Helm charts in the other PR.